proto/nexus/v1beta1/query.proto

```diff
@@ -22,3 +23,15 @@ message LatestDepositAddressRequest {
 }
 
 message LatestDepositAddressResponse { string deposit_addr = 1; };
+
+// TransfersForChainRequest represents a message that queries the
+// transfers for the specified chain
+message TransfersForChainRequest {
+  string chain = 1;
+  exported.v1beta1.TransferState state = 2;
```

Comment:
The reason why I originally used a string instead of the actual type was because of the rest RPC endpoint. If we go about it like this, we will need to pass an integer value as a parameter instead of a string. I don't think that is very user friendly.

Reply:
we don't if we don't translate the request into path parameters (which doesn't make sense with the pagination anymore anyway)